allianceutils/auth/models.py

```diff
@@ -248,8 +248,7 @@ def normalize_email(cls, email):
         return email.lower()
 
     def save(self, *args, **kwargs):
-        if not self.pk and self.__class__.objects.filter(email=self.__class__.normalize_email(self.email)).exists():
-            raise ValueError('Sorry, this email is not available.')
```

Comment:
Is this check handled somewhere else? eg. unique validation 

if not we should have this in somewhere still (probably `_validate_email` that's called in `clean` and in `save`)

Reply:
it'll trigger an IntegrityError (see test case) on save - django's internal validation to stop same users w/ duplicated username from being created (edited)